_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,175 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any specific display orientation using CSS media queries, and the content is operable in all display orientations.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any `element` that is [visible](#visible) and has a CSS `transform` property with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) transform-function, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`landscape` or `portrait`).
+
+## Expectation
+
+Each target element does not restrict its view and operation to a single display orientation, unless a specific display orientation is [essential](https://www.w3.org/TR/WCAG21/#dfn-essential).
```

Comment:
> Each target element does not restrict its view and operation to a single display orientation

This is ambiguous. That's not allowed in the ACT Rules Format. You're going to have to define these terms if you want to use them.